### DISCUSSION TITLE
predictionCutoff parameter in addPeak2GeneLinks

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#1814] [Questions / Documentation]
Hi @rcorces and @jeffmgranja ,

I was looking at the source code for `addPeak2GeneLinks` and the use of predictionCutoff:

https://github.com/GreenleafLab/ArchR/blob/f6c0388bd37023400794c9ae8562ad69e3ba9fd7/R/IntegrativeAnalysis.R#L1035-L1057

It appears that the predictionCutoff is used to subset `dfAll` but is not used later on when making metacells. The object `dfAll` is also not used later on. If I am misinterpreting, if you could point me in the direction of where predictionCutoff is used to subset out cells, that would be greatly appreciated! Thank you for your help.

**Comment by rcorces** [comment]
Thanks for posting this and sorry it has taken me a while to look into this. I see exactly what you see. I imagine this is a bug but will have to look into it more. I'm migrating this to an Issue for tracking purposes.

**Reply by RegnerM2015** [reply]
No worries! Thanks for checking into it. 

It could be convenient to create the subset of cells meeting the predictionCutoff here (similar to cellsToUse):

https://github.com/GreenleafLab/ArchR/blob/f6c0388bd37023400794c9ae8562ad69e3ba9fd7/R/IntegrativeAnalysis.R#L1070-L1074
